src/components/PolarisTestProvider/PolarisTestProvider.tsx

```diff
@@ -108,9 +122,13 @@ export function PolarisTestProvider({
                   <LinkContext.Provider value={link}>
                     <ThemeContext.Provider value={mergedTheme}>
                       <MediaQueryContext.Provider value={mergedMediaQuery}>
-                        <FrameContext.Provider value={mergedFrame}>
-                          {children}
-                        </FrameContext.Provider>
+                        <FocusManagerContext.Provider
+                          value={mergedFocusManager}
+                        >
```

Comment:
Should we just use `FocusManager` and not allow any overrides?

Reply:
I think for now we probably should. Not sure we'll ever need to override it. We can do it when we do.

Reply:
It's a different pattern than we've previously been using, however `FocusManager` has some overhead we would need to replicate in mocks and it's easier to use the component. Since this is a public API let's go with using FocusManager since adding to an API is easier than removing.